### PR TITLE
Filter tab

### PR DESCRIPTION
**Pull request by BastienSaulnier**
Filter Tab Component

### CODE REVIEW

File: frontend/src/components/FilterTab.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./styles/FilterTab.scss";
+
+function FilterTab() {
+  return (
+    <div className="FilterTab">
+      <div className="center">
+        {/* GET CONCERNED TITLE TO INSERT IT THERE ( AMBASSADORS? CAUSES? ) */}
+        <p>Filtrer parmis les /* import name props */:</p>
```
Comment (TheoBoschetti) [inline frontend/src/components/FilterTab.jsx]: A variable can bu put here  (props)

**Review by TheoBoschetti — COMMENTED** [review]
ok

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Comment by TheoBoschetti** [comment]
the component is not responsive, and has to be remade/redesigned imo

**Review by LeoDiduch — DISMISSED** [review]
OK

File: frontend/src/components/FilterTab.jsx (hunk 1 above)
Comment (LoicBrassart) [inline frontend/src/components/FilterTab.jsx]: I think you didn't finish this line of code

File: frontend/src/components/FilterTab.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <nav className="FiltersNav">
+          <ul>
+            {/* USE .map TO GENERATE BUTTONS ACCORDDING TO THE NUMBER OF CATEGORIES INVOLVED*/}
```
Comment (LoicBrassart) [inline frontend/src/components/FilterTab.jsx]: Please don't include "TODO" instructions: either you do it, or you don't, but don't "hint" like that

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Things to change

File: frontend/src/components/FilterTab.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/components/FilterTab.jsx]: Done !

File: frontend/src/components/FilterTab.jsx (hunk 2 above)
Comment (BastienSaulnier) [reply frontend/src/components/FilterTab.jsx]: Done !

**Review by LeoDiduch — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**